css/components/_passage.css

```diff
@@ -120,3 +120,34 @@
   }
 
 }
+
+.reciteMode {
+  text-align: center;
+
+  .actions {
+    display: flex;
+    justify-content: space-around;;
+
+    button {
+      background-color: $vintage-eggy;
```

Comment:
I feel like switching up the colors for these buttons might be a good idea? Consider `$vintage-slimer`?